src/atopile/buildutil.py

```diff
@@ -21,6 +25,7 @@
     export_pick_and_place,
     export_step,
 )
+from faebryk.exporters.pcb.kicad.pcb import _get_footprint
```

Comment:
Used as a temp measure to check footprint quality early and avoid late (eg. long) expensive failures